sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/EventGridPublisherClient.java

```diff
@@ -38,8 +41,8 @@ public EventGridServiceVersion getServiceVersion() {
      * @param events the EventGrid events to publish.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void sendEvents(Iterable<EventGridEvent> events) {
-        asyncClient.sendEvents(events, Context.NONE).block();
+    public void sendEventGridEvents(Iterable<EventGridEvent> events) {
+        asyncClient.sendEventGridEvents(events, Context.NONE).block();
```

Comment:
add `@throws` clause in all the sync API also .

Reply:
Added

Reply:
Added